recipes/ace_cream/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ace_cream" %}
+{% set version = "0.4.post8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/zhaofeng-shu33/ace_cream/archive/v{{ version }}.tar.gz
+  sha256: f6fc27061e3a33edc3e47a6b1d67d2d14a6cd7cb90671aea2f4dbf1605f64a61
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('fortran') }}
+  host:
+    - python
+    - numpy
```

Comment:
Please see the following about building against numpy:

https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
> Please see the following about building against numpy:
> 
> https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

I have used `pin_compatible`. However, the build for link still pops up `Package python conflicts`